### ISSUE TITLE
Central articles overflow to right

### ISSUE THREAD

**Issue by kewbish** [thepracticaldev/dev.to#8278]
**Describe the bug**
The central articles column overflows completely to the right, without an overflow scroll bar.  
This hides the right sidebar completely, and most of the article text is also obscured.

The page seems to load initially fine, with the skeleton loaders in the correct central position and not overflowing. However, when articles load in, the central columns seems to overflow, with article sizes increasing as well.

Error usually persists, but sometimes the central column also loads properly.

**To Reproduce**
1. Load Dev.to page
2. See error

**Expected behavior**
Central article columns remains in the center and does not overflow.

**Screenshots**

![Screenshot (5)](https://user-images.githubusercontent.com/45278276/83787777-23b35480-a649-11ea-80eb-a71f65670ab3.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome
- Version: v83

**Smartphone (please complete the following information):**
Seems to work fine on my phone.

- Device: - 
- OS: - 
- Browser: -
- Version: -

**Additional context**
Bug is reproducible, but not 100% of the time.

**Comment by kewbish** [comment]
Update, it seems to be regularly fitting back in the center now. Wonder why this happens some of the time and not all of the time.

**Comment by ludwiczakpawel** [comment]
hey @kewbish thanks for the report. 

> Update, it seems to be regularly fitting back in the center now.

We noticed the issue pretty quickly so we pushed quick hack to fix it temporarily.

> Wonder why this happens some of the time and not all of the time.

It was caused by the code blocks in comments, in particular `.highlight` element is weirdly behaving, probably because of its nature - it contains `pre` element which itself is a bit weird in terms of rendering. And you won't always see comments with code blocks in the feed so that's why it sometimes happens and sometimes don't. It (feed) could also have different width depending on how long the code line was so yea... Just a tricky thing to test locally :D
